pubspec.yaml

```diff
@@ -7,7 +7,7 @@ name: stack_trace
 #
 # When the major version is upgraded, you *must* update that version constraint
 # in pub to stay in sync with this.
-version: 1.8.1
+version: 1.8.2
```

Comment:
Does this use the new API? Need to bump SDK constraint?

Reply:
Instead, do `1.8.2-dev` for now. Use that DEP in the SDK. We can talk about releasing when we're ready to publish that SDK

Reply:
done.